CHANGELOG.md

```diff
@@ -23,11 +25,24 @@
 - `az_iot_provisioning_client_init()` parameter `global_device_endpoint` renamed to `global_device_hostname`.
 - Renamed the macro `AZ_SPAN_NULL` to `AZ_SPAN_EMPTY`.
 - Renamed the `az_result` value `AZ_ERROR_INSUFFICIENT_SPAN_SIZE` to `AZ_ERROR_NOT_ENOUGH_SPACE`.
+- Made the helper macros `AZ_RETURN_IF_FAILED` and `AZ_RETURN_IF_NOT_ENOUGH_SIZE` internal.
+- Behavioral change to disallow passing `NULL` to `az_context` and update documentation.
+- Removed `AZ_HUB_CLIENT_DEFAULT_MQTT_TELEMETRY_DUPLICATE` and `AZ_HUB_CLIENT_DEFAULT_MQTT_TELEMETRY_RETAIN` from `az_iot_hub_client.h`.
 
 ### Bug Fixes
 
+- Fix the strict-aliasing issue in `_az_isfinite()`.
+- Fix the SDK warnings for the release configurations.
+- Do not use a shared static scratch buffer for JSON token parsing. Instead use stack space.
+
 ### Other Changes and Improvements
 
+- Refactor and update IoT samples.
+- Add support to HTTP request set query parameter for customers to tell if the values are are already URL encoded.
+- Optimize for code size URL encoding, appending path to request, and setting query parameters.
+- Add ARM to the CI matrix.
+- Add map file generation to CI.
```

Comment:
Last 2 items: does CI go to log?
Instead of saying that we added ARM to CI matrix, should we now instead say say that we support ARM?

Reply:
Will update. @danieljurek - any thoughts on this?

Reply:
> should we now instead say say that we support ARM?

I don't want to say that per-se, because we have always supported ARM, and didn't really make much changes in this release particularly to enable that.